pkg/ride/tree_estimatorV1.go

```diff
@@ -103,6 +104,8 @@ func newTreeEstimatorV1(tree *ast.Tree) (*treeEstimatorV1, error) {
 		r.scope = newEstimationScopeV1(ConstantsV6, CatalogueV6)
 	case ast.LibV7:
 		r.scope = newEstimationScopeV1(ConstantsV7, CatalogueV7)
+	case ast.LibV8:
+		r.scope = newEstimationScopeV1(ConstantsV8, CatalogueV8)
```

Comment:
Why we need to handle RideV8 scripts for the first version of estimator when version 4 will be used for RideV8 scripts? Can be applied for other estimators (excluding version 4).

If this is intended logic, please, explain why

Reply:
The estimation logic made independent of features activation. That's it.